modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -10,7 +10,7 @@ import edu.gemini.epics.acm.CaService
 import edu.gemini.pot.sp.SPObservationID
 import edu.gemini.spModel.core.Site
 import edu.gemini.seqexec.engine
-import edu.gemini.seqexec.engine.{Action, Engine, Event, EventSystem, Executed, Failed, Sequence}
+import edu.gemini.seqexec.engine.{Step => _, _}
```

Comment:
Did not know about `import ... A => _`
I suppose that means ignore `Step` in the package import, because you're using `Step` as defined by another import?

Reply:
That was Idea's magic. I thing it reduced all the imports from `engine` to all minus Step.

Reply:
Interesting. Idea does some very funny things sometimes.